### PR TITLE
editorconfig-checker: init at 2.0.2

### PR DESCRIPTION
**Pull request by uri-canva**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @dochang @FRidh @costrouc

### CODE REVIEW

File: pkgs/development/tools/misc/editorconfig-checker/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    rev = "${version}";
+    sha256 = "0v2ml9r8b5admi3sv80wa1pwl9qnz03q2p84vgcmgg2nv1v6yxf3";
+  };
+
+  modSha256 = "09b1v9gyh6827yqlfxxxq3lcqhd5snn3n7gdlbjmga3wyp2x4g2r";
+
+  meta = with lib; {
+    description = "A tool to verify that your files are in harmony with your .editorconfig";
+    homepage = "https://editorconfig-checker.github.io/";
+    license = lib.licenses.mit;
```
Comment (jonringer) [inline pkgs/development/tools/misc/editorconfig-checker/default.nix]:
```suggestion
    license = licenses.mit;
```

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM (1 minor suggestion)
binary seems to worok
leaf package

**Comment by uri-canva** [comment]
ping @jonringer

**Comment by uri-canva** [comment]
ping @dochang @FRidh @costrouc @kalbasit

**Comment by jonringer** [comment]
I'm in the same boat, i got 25 open PRs :(

**Pull request merged**